bot/exts/halloween/hacktoberstats.py

```diff
@@ -157,7 +157,7 @@ async def build_embed(self, github_username: str, prs: List[dict]) -> discord.Em
 
         stats_embed = discord.Embed(
             title=f"{github_username}'s Hacktoberfest",
-            color=discord.Color(0x9c4af7),
+            color=0x9c4af7,
```

Comment:
This should be promoted to a constant.

Reply:
I'll just use [`Colours.purple`](https://github.com/python-discord/sir-lancebot/blob/main/bot/constants.py#L146).

Reply:
Also resolved in https://github.com/python-discord/sir-lancebot/pull/718/commits/c051abf49ce5bb82d127c8ddfad7ece431bb028f.